File: TellDontAsk/AlarmsObjectOriented/Criterias/DayCriteria.cs
using System;
using AlarmsObjectOriented.Interfaces;
using TddEbook.TddToolkit;

namespace AlarmsObjectOriented.Criterias
{
  internal class DayCriteria : TimeCriteria
  {
    public bool IsSatisfied()
    {
      return Any.Boolean();
    }

    public void Output()
    {
      Console.WriteLine("it's a day");
    }
  }
}

File: Java/AlarmsObjectOriented/src/main/java/criterias/DayCriteria.java
package criterias;

import interfaces.TimeCriteria;

import static autofixture.publicinterface.Generate.any;

public class DayCriteria implements TimeCriteria {
  public boolean isSatisfied() {
    return any(Boolean.class);
  }

  public void output() {
    System.out.println("it's a day");
  }
}
